Creates test scenarios and test zones for test cases Delegation01, -02 and -03 (#1305)
zonemaster/zonemaster: docs/public/specifications/test-zones/README.md

requirements for creating separate zones for `in-addr.arpa` or `ip6.arpa` or
below.

### Undelegated data

> Some test scenarios require that the test zones is resolved as
> [undelegated test]. In that case, the scenario specification will contain a small

Reviewer: ```suggestion
Some test scenarios require that an [undelegated test] be carried out on it.
In that case, the scenario specification will contain a small
```
